api/src/dto/ContentDto.ts

```diff
@@ -1,36 +1,60 @@
-import { IsArray, IsNotEmpty, IsNumber, IsString } from "class-validator";
-import { ContentStatus, DocType, Uuid } from "../types";
+import { IsDate, IsEnum, IsNotEmpty, IsOptional, IsString } from "class-validator";
+import { ContentStatus, Uuid } from "../enums";
+import { _contentBaseDto } from "./_contentBaseDto";
 
 /**
  * Database structured Content object
  */
-export class ContentDto {
-    @IsNotEmpty()
-    _id: Uuid;
-    @IsNotEmpty()
-    type: DocType.Content;
-    @IsArray()
-    memberOf: Uuid[];
+export class ContentDto extends _contentBaseDto {
     @IsNotEmpty()
+    @IsString()
     language: Uuid;
+
     @IsNotEmpty()
+    @IsEnum(ContentStatus)
     status: ContentStatus;
+
     @IsString()
+    @IsNotEmpty()
     slug: string;
+
     @IsString()
+    @IsNotEmpty()
     title: string;
+
     @IsString()
+    @IsNotEmpty()
     summary: string;
+
     @IsString()
+    @IsOptional()
```

Comment:
For readability (but perhaps it's a hassle to change it now) I would propose to put `IsOptional` at the top, so it's easier to spot when a field it optional.

Reply:
Agree